### PR TITLE
Fix  `tns run ios --justlaunch`

### PR DESCRIPTION
**Pull request by Fatme**
In case when `tns run ios --justlaunch` command is executed on real iOS device, the console should be released and the process should exit. Actually the process does not exit because a device's log process is started and this prevents the console to be released. So this PR fixes this behaviour.

Fixes https://github.com/NativeScript/nativescript-cli/issues/3644

## PR Checklist

- [ ] The PR title follows our guidelines: https://github.com/NativeScript/NativeScript/blob/master/CONTRIBUTING.md#commit-messages.
- [ ] There is an issue for the bug/feature this PR is for. To avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it.
- [ ] You have signed the [CLA](http://www.nativescript.org/cla).
- [ ] All existing tests are passing: https://github.com/NativeScript/nativescript-cli/blob/master/CONTRIBUTING.md#contribute-to-the-code-base
- [ ] Tests for the changes are included.

## What is the current behavior?
The console is not released when `tns run ios --justlaunch` command is executed.

## What is the new behavior?
The console is released when `tns run ios` command is executed.

### CODE REVIEW

File: lib/services/debug-service.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ export class DebugService extends EventEmitter implements IDebugService {
 			this.$errors.failWithoutHelp(`Unsupported device OS: ${device.deviceInfo.platform}. You can debug your applications only on iOS or Android.`);
 		}
 
-		if (this.$mobileHelper.isiOSPlatform(device.deviceInfo.platform)) {
+		if (this.$mobileHelper.isiOSPlatform(device.deviceInfo.platform)) { // TODO: Consider to move this code to ios-debug-service
```
Comment (rosen-vladimirov) [inline lib/services/debug-service.ts]: Please move the TODO above the code

File: lib/services/livesync/livesync-service.ts [hunk 2]
```diff
@@ -456,8 +456,11 @@ export class LiveSyncService extends EventEmitter implements IDebugLiveSyncServi
 		const deviceAction = async (device: Mobile.IDevice): Promise<void> => {
 			try {
 				const platform = device.deviceInfo.platform;
+				const liveSyncService = this.getLiveSyncService(platform);
```
Comment (rosen-vladimirov) [inline lib/services/livesync/livesync-service.ts]: this should be called `const platformLiveSyncService`

**Comment by miroslavaivanova** [comment]
run ci

**Comment by miroslavaivanova** [comment]
run ci

**Pull request merged**